### DISCUSSION TITLE
Here is a tool I created to split big .docx files like a book with chapters, to use with Library

### DISCUSSION THREAD

**Discussion by midorikocak** [nytimes/library#262] [Show and tell]
Not a feature request but I would like to get your opinions.

http://github.com/midorikocak/docsplitter

### Problem Description
For a project, I created for a client, the content and sections were saved in large .docx files with small chapters. As I know, Library requires the google doc files to be separated. To split many large .docx files at least 36 sections would be a tedious task. I looked for a tool to split .docx files according to headings but I could not find one. Using master documents and generate chapter documents one by one was also confusing. So I decided to use python-docx library to iterate and generate zipped .docx files with chapter numbering.

### Feature

Suppose you have `book with long chapters.docx` file. 

``` bash
$ docsplitter -f  book\ with\ long\ chapters.docx
```

will generate `book with long chapters.zip` in the folder you run the docsplitter. When you extract the zipfile splitted files should appear like `1 - book with long chapters - Chapter Name.docx`

If you don't want the filename as prefix you can use `-nn, --noname` option.

``` bash
$ docsplitter -f  book\ with\ long\ chapters.docx -nn
```

The generated files will not include the original filename and will have only headings. `1 - Chapter Name.docx`

If you want to split your word documents using a different heading, you can use   `-l, --level` option.

### Additional Information
It would be great if you give feedback on this tool, what could be improved for the compatibility. 

Thank you very much.

**Comment by patcon** [comment]
I'm not active on this project, but just wanted to say that it's really generous of you to upstream this. Thanks!
